assign.py

```diff
@@ -272,6 +274,17 @@ def main():
             else:
                 procversion = wf_info["ProcessingVersion"]
 
+        # reading xrootd and secondary_xrootd values
```

Comment:
the issue with doing so @prozober is that you have no way at all to tell "trust_site=False" for a recovery that was ran with trust_site=True. you would need a tri-valued --xrootd option that can take
--xrootd ==> True
--xrootd False ==> False
or a --no_xrootd and --no_secondary_xrootd